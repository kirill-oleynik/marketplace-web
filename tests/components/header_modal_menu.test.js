const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const {
  HeaderModalMenu
} = require('../../app/components/header_modal_menu');

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <HeaderModalMenu
        isOpen={false}
        toggle={() => {}}
        currentUser={{}}
        t={() => {}}
        openProfile={() => {}}
      />
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
