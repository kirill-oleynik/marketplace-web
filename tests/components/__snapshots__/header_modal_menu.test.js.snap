// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<Modal
  autoFocus={true}
  backdrop={false}
  backdropTransitionTimeout={150}
  className="header-modal"
  fade={true}
  isOpen={false}
  keyboard={true}
  modalTransitionTimeout={300}
  toggle={[Function]}
  zIndex={1050}
>
  <Container
    tag="div"
  >
    <ModalHeader
      tag="h4"
      toggle={[Function]}
      wrapTag="div"
    />
    <div>
      <div>
        <Link
          href="/sign_in"
        >
          <a
            className="header-modal__link"
          />
        </Link>
        <Link
          href="/sign_up"
        >
          <a
            className="header-modal__link"
          />
        </Link>
      </div>
    </div>
  </Container>
</Modal>
`;
